examples/jsm/exporters/USDZExporter.js

```diff
@@ -445,23 +439,40 @@ function buildMaterial( material ) {
 
 	if ( material.aoMap !== null ) {
 
```

Comment:
AO map is enabled now, but it will be broken if you actually use tiling on diffuse map other than (1,1). I was also thinking about introducing options parameter for the parse() function (similarly to how it is for export glTF). One can specify maps to exclude (or include) in options in order to avoid using AO map where tiling can break it.